src/model/RaceWinners.js

```diff
@@ -0,0 +1,66 @@
+import { ERROR_MESSAGE } from '../constants/errorMessages.js'
+
+export class RaceWinners {
+  #winners
+
+  constructor(records) {
+    this.#validateRecords(records)
+    this.#winners = this.#findWinners(records)
+  }
+
+  #validateRecords(records) {
+    if (!this.#isRecordsExist(records) || !this.#isRecordValuesValid(records)) {
+      throw new Error(ERROR_MESSAGE.COLLECTIVE_RECORDS_FORMAT)
+    }
+  }
+
+  #isRecordsExist(records) {
+    return records && typeof records === 'object' && !Array.isArray(records)
+  }
+
+  #isRecordValuesValid(records) {
+    const values = Object.values(records)
+
+    // 빈 value 인지 확인
+    if (!values.length) return false
+
+    // 모든 values 의 길이가 동일한지 확인
+    const lengths = values.map((value) => value.length)
+    const uniqueLengths = [...new Set(lengths)]
+    if (uniqueLengths.length !== 1) return false
+
+    // 모든 values 가 배열인지 확인
+    if (!values.every((value) => Array.isArray(value))) return false
+
+    // 모든 values 의 value 요소가 0이상의 숫자인지 확인
+    if (
+      !values
+        .flatMap((value) => value)
+        .every((value) => typeof value === 'number' && value >= 0)
+    )
+      return false
+
+    return true
```

Comment:
<img width="241" alt="image" src="https://github.com/next-step/js-racingcar/assets/18749057/0404f374-7363-4494-a3ac-fadeb528fd72">

```suggestion
    return values.flat().every((value) => typeof value === 'number' && value >= 0)
```

이렇게 표현할수 있겠네요!